### ISSUE TITLE
Improve home page's keyboard navigation 

### ISSUE THREAD

**Issue by karatechops** [hpe-design/design-system#257]
The current tab index flow:
- Dark toggle
- Header logo
- Search box
- Footer items

Users should be able to navigate the homepage tiles with tab index. This probably relates to #256 as the browser is probably looking for an `a` or `button` tag to focus.

**Comment by taysea** [comment]
The tabbing behavior with the homepage tiles is actually an issue with how grommet modulates tab index when drop opens. Eric and I worked on it, so that piece should be fixed once this PR merges into grommet:
https://github.com/grommet/grommet/pull/3650

**Comment by MikeWalrath** [comment]
@taysea the Grommet pr went in. Can we verify that this is now fixed?

**Comment by taysea** [comment]
Yes, closing now.
